guidelines/enhancement_template.md

```diff
@@ -0,0 +1,282 @@
+---
+title: neat-enhancement-idea
+authors:
+  - "@janedoe"
+reviewers:
+  - TBD
+  - "@alicedoe"
+approvers:
+  - TBD
+  - "@oscardoe"
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced|informational
+see-also:
+  - "/enhancements/this-other-neat-thing.md"  
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+Start by filling out this header template with metadata for this enhancement.
+
+* `reviewers`: This can be anyone that has an interest in this work.
+
+* `approvers`: All enhancements must be approved, but the appropriate people to
+  approve a given enhancement depends on its scope.  If an enhancement is
+  limited in scope to a given team or component, then a peer or lead on that
+  team or pillar is an appropriate approver.  If an enhancement captures
+  something more broad in scope, then a member of the OpenShift architects team
+  or someone they delegate would be appropriate.  Examples would be something
+  that changes the definition of OpenShift in some way, adds a new required
+  dependency, or changes the way customers are supported.  Use your best
```

Comment:
i think we should do a little more to spell this out. which part of the OCM community needs to approve stuff?

Reply:
I'm thinking it over. Any comments @deads2k @mdelder

Reply:
I think for now the best play is probably to make the equivalent of sig arch. I'd avoid makin a lot of sigs without a lot of external participation.